radio/src/targets/horus/hal.h

```diff
@@ -245,8 +245,8 @@
     #define SWITCHES_E_INVERTED
   #endif
 
-  #if defined(PCBX10)
-  // Gimbal switch left
+  #if defined(PCBX10) && !defined(RADIO_FAMILY_T16)
+    // Gimbal switch left
```

Comment:
Hm... TX16S is in that family also.

![image](https://github.com/EdgeTX/edgetx/assets/5500713/00e462fd-a89d-459b-af11-27036c802049)